src/pmdas/openmetrics/pmdaopenmetrics.python

```diff
@@ -685,6 +686,30 @@ class Source(object):
             self.pmda.debug("included_labels '%s'" % (included_labels)) if self.pmda.dbg else None
             self.pmda.debug("optional_labels '%s'" % (optional_labels)) if self.pmda.dbg else None
             if sp.name in self.metrics_by_name:
+                # re-add metric to namespace
+                if self.name in self.pmda.re_add_list:
+                    if pcpline:
+                        split = pcpline.split(" ")
+                        fullname = "openmetrics.%s.%s" % (self.name, split[1])
+                    else:
+                        fullname = "openmetrics.%s.%s" % (self.name, sp.name.replace(":", "."))
+                    if helpline: # it could be None!
+                        unescaped = helpline.replace('\\\\', '\\').replace('\\n', '\n')
+                        split = unescaped.split('\n')
+                        help_oneline = split[0] # must have at least one entry
+                        help_text = '\n'.join(split[1:]) # may have other entries
+                    else:
+                        help_oneline = ''
+                        help_text = ''
```

Comment:
The above 8 lines dealing with help text extraction / escaping are duplicated from line ~130 - should be a shared function implementing this now.  Input the (helpline) string, pass out a tuple (oneline, text).